documentation/modules/upgrading/proc-reenabling-FIPS-mode.adoc

```diff
@@ -0,0 +1,40 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='proc-reenabling-FIPS-mode-{context}']
+= Re-enabling FIPS mode
+
+[role="_abstract"]
+Until Strimzi 0.33, running on FIPS-enabled Apache Kafka clusters was possible only with the FIPS mode disabled using the `FIPS_MODE` environment variable.
+But from 0.33 on, Strimzi supports running on FIPS-enabled Kubernetes clusters without disabling the FIPS mode.
+If you run Strimzi on a FIPS-enabled Kubernetes cluster with the `FIPS_MODE` set to `disabled`, you can re-enable it by following this procedure.
+
+.Prerequisites
+
+* FIPS-enabled Kubernetes cluster
+* An existing Cluster Operator deployment with the `FIPS_MODE` environment variable set to `disabled`
+
+.Procedure
+
+. Upgrade Strimzi Cluster Operator to version 0.33 or newer but keep the `FIPS_MODE` environment variable set to `disabled`
+
+. If you initially deployed your cluster with a Strimzi version older than 0.30, it might still use the PKCS12 stores using old encryption and digest algorithms which are not supported with FIPS enabled.
```

Comment:
```suggestion
. If you initially deployed a Strimzi version older than 0.30, it might use old encryption and digest algorithms in its PKCS #12 stores, which are not supported with FIPS enabled.
```